index.d.ts

```diff
@@ -866,6 +867,16 @@ declare namespace Eris {
     selfStream: boolean;
     selfVideo: boolean;
   }
+  interface VoiceChannelEffect {
+    guild: Guild | Uncached;
+    channel: AnyVoiceChannel | Uncached;
+    user: User | Uncached;
+    emoji?: PartialEmoji;
+    animationType?: VoiceChannelEffectAnimationType;
+    animationID?: number;
+    soundID?: string | number;
+    soundVolume?: number;
+  }
```

Comment:
Needs to be alphabetically sorted, also placed under the Voice category, not Events